test.py

```diff
@@ -459,6 +459,20 @@ def test(bad, good):
         yield test, bad, good
 
 
+def test_remove_default_port():
+    def test(query, result):
+        assert_equal(url.parse(query).remove_default_port().utf8(), result)
+
+    examples = [
+        ('http://foo.com:80/'  , 'http://foo.com/'     ),
+        ('https://foo.com:443/', 'https://foo.com/'    ),
+        ('http://foo.com:8080/', 'http://foo.com:8080/')
```

Comment:
Maybe add a negative case for `https` as well.

You might also want to add cases for a relative URL or a scheme-relative URL like: `//foo.com/:80/`.